digitalfabrik/entitlementcard
# 1561: Add import store role

### Short description


In preparation of #1475 we need a separate role for importing accepting stores.

### Proposed changes



- add the role PROJECT_STORE_MANAGER
- add runConfig to create a user for nuernberg
- adjust constraint for admin table
- add route with placeholder content
- add navigation and dashboard item
- add backend test for Authorizer

### Side effects



- none

### Resolved issues



Fixes: #1561

### Testing (dev only)
- Migrate DB
- Create a Project Store Manager user
- run backend service
- Switch to Nuernberg and Login
- Check if route `/stores` can be accessed
